testutils/testutils.go

```diff
@@ -13,3 +16,54 @@ func NewTestLogger() (*zap.Logger, error) {
 
 	return loggerConfig.Build()
 }
+
+func NewTestSugaredLogger() (*zap.SugaredLogger, error) {
```

Comment:
This is inside the `testutils` package, the word `test` sounds redundant:

`NewTestSugaredLogger` -> `NewSugaredLogger`

Reply:
updated